content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -228,31 +270,35 @@ git checkout master
 
 Alternatively, you can download an archive (.zip or .tar.gz) of a specific release from the [RippleAPI releases page](https://github.com/ripple/ripple-lib/releases) and extract it.
 
-#### 2. Install dependencies using NPM
 
-You need to have [NPM (Node.js Package Manager) installed](#install-nodejs-and-npm) first.
+### 2. Install Yarn
+
+Use these instructions to [install Yarn](#install-yarn).
+
 
-Then, from within the `ripple-lib` directory, you can use NPM to install all the necessary dependencies:
+### 3. Install dependencies using Yarn
 
 ```
-npm install
+yarn
 ```
 
-(We recommend _not_ using `npm -g` to install dependencies globally.)
-
-This can take a while, and may include some warnings. The following warnings are benign and do not indicate a problem:
+The install process may end with a few warnings. You may safely ignore the following warnings: ***TODO: necessary to mention? Existing doc surfaced NPM warnings - so thought I'd do the same for yarn?***
```

Comment:
Personally I don't think it's necessary but I'm fine either way!

Reply:
Removed it.